src/Arcus.Messaging.Tests.Unit/ServiceBus/MessageExtensionsTests.cs

```diff
@@ -94,6 +94,27 @@ public void GetMessageCorrelationInfo_WithTransactionIdAsUserProperty_ReturnsCor
             Assert.Equal(expectedTransactionId, correlationInfo.TransactionId);
         }
 
+        [Fact]
+        public void GetMessageCorrelationInfo_WithTransactionIdInCustomUserProperty_ReturnsCorrelationInfoWithGeneratedOperationId()
```

Comment:
An test where we get the custom transaction ID directly can be appropriate also.

Reply:
What would that be then? Not sure if I get the suggestion.

Reply:
Well, there was two methods updated but only one updated method is tested here, right? Or am I wrong?

Reply:
We don't have tests for the other one but will add thme